src/market/market.py

```diff
@@ -163,15 +171,40 @@ def place_order(  # pylint: disable=too-many-arguments
 
         if payer == owner.public_key():
             raise ValueError("Invalid payer account")
+
+        should_wrap_sol = False
+        wrapped_sol_account = Account()
         if (side == side.Buy and self.state.quote_mint() == WRAPPED_SOL_MINT) or (
             side == side.Sell and self.state.base_mint == WRAPPED_SOL_MINT
         ):
-            # TODO: Handle wrapped sol account
-            raise NotImplementedError("WRAPPED_SOL_MINT is currently unsupported")
+            should_wrap_sol = True
+            transaction.add(
+                create_account(
+                    CreateAccountParams(
+                        from_pubkey=owner.public_key(),
+                        new_account_pubkey=wrapped_sol_account.public_key(),
+                        lamports=Market.get_lamport_need_for_sol_wrapping(
+                            limit_price, max_quantity, side, open_order_accounts
+                        ),
+                        space=165,
```

Comment:
Use this constant
https://github.com/michaelhly/solana-py/blob/master/spl/token/constants.py#L8

Reply:
done.